### PR TITLE
Reviewed slices.

### PR DESCRIPTION
**Pull request by wayneeseguin**
Reviewed the slices chapter. Still loving the book :)

### CODE REVIEW

**Comment by initpy** [comment]
Phew, I'm speechless! You really did a wonderful job! It's _me_ who's loving what you're doing here. You sliced my humble work, and made it grow in beauty :)

File: slices.rst [hunk 1]
```diff
@@ -438,40 +450,40 @@ Output:
...
+Do you see the difference? ``make`` doesn't merely allocate enough space to
+*host* the structure of a given slice (the pointer, the length and the capacity)
+only, it allocates both the structure *and* the underlying array.
 
 Back to our resizability of slices. As you may guess, shrinking a slice is
-really easy! All we have to do is reslice it and assign the new slice to it. 
+really easy! All we have to do is reslice it and assign the new slice to it.
 
-.. insert example of shrinking a slice here
+.. insert example of shrinking a slice here -- Shrinkage!!! Cold water Example!!!
```
Comment (initpy) [inline slices.rst:481]: :D

Comment (wayneeseguin) [inline slices.rst:481]: Sorry, couldn't help myself ;) Plus I figured that didn't generate onto the website ;)

Comment (initpy) [inline slices.rst:481]: It made me laugh, and that itself is a _big_ win! Kudos! :)

**Pull request merged**